camera_params: 
  camera_resolution: HD1080
  depth_mode: PERFORMANCE
  unit: METER
  sdk_verbose: 1

tracking_params:
  detection_model: HUMAN_BODY_FAST
  enable_tracking: True
  image_sync: True
  enable_segmentation: False
  enable_body_fitting: True
  set_as_static: True
  set_floor_as_origin: True
  detection_confidence_threshold: 40

ros_params:
  node_name: zed_tracking_node